src/views/HeaderBackButton.js

```diff
@@ -27,6 +29,11 @@ const HeaderBackButton = ({ onPress, tintColor }: Props) => (
       source={require('./assets/back-icon.png')}
       tintColor={tintColor}
     />
+    {Platform.OS === 'ios' && title && (
+      <Text style={{ color: tintColor }}>
```

Comment:
Probably this needs additional styles to match the native look? Also the default tint color should be the same as the back button.